pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -71,19 +71,19 @@ type NodeLabeller struct {
 	queue                   workqueue.RateLimitingInterface
 	supportedFeatures       []string
 	cpuModelVendor          string
-	volumePath              string
+	volumePath		string
 	domCapabilitiesFileName string
-	capabilities            *libvirtxml.Caps
+	cpuCounter		*libvirtxml.CapsHostCPUCounter
 	hostCPUModel            hostCPUModel
 	SEV                     SEVConfiguration
 	arch                    string
 }
 
-func NewNodeLabeller(clusterConfig *virtconfig.ClusterConfig, nodeClient k8scli.NodeInterface, host string, recorder record.EventRecorder) (*NodeLabeller, error) {
-	return newNodeLabeller(clusterConfig, nodeClient, host, nodeLabellerVolumePath, recorder)
+func NewNodeLabeller(clusterConfig *virtconfig.ClusterConfig, nodeClient k8scli.NodeInterface, host string, cpuCounter *libvirtxml.CapsHostCPUCounter, recorder record.EventRecorder) (*NodeLabeller, error) {
```

Comment:
nit: Please consider adding the `cpuCounter` as the last parameter.
It is reasonable to assume that follow-up PRs will extract additional file-system reads and add additional parameters here (or a builder).

Reply:
Done.